classification/templates/classification/classification.html

```diff
@@ -40,7 +41,9 @@
                     reportEnabled: {{ report_enabled | jsonify }},
                     deleteEnabled: {{ delete_enabled | jsonify }},
                     attachmentsEnabled: {{ attachments_enabled | jsonify }},
-                    withdrawReasons: {{ withdraw_reasons | jsonify }}
+                    withdrawReasons: {{ withdraw_reasons | jsonify }},
+                    conditionresolution: {{ conditiontextmatch.pk | jsonify }},
+                    condition_matching_is_view_enabled: "{{ "condition_matching" | is_view_enabled | jsonify }}",
```

Comment:
No need to turn this into a string, if you remove the double quotes around this
`"{{ "condition_matching" | is_view_enabled | jsonify }}"`
you'll just get a JavaScript boolean